src/Kip.scss

```diff
@@ -215,7 +215,11 @@
       width: calc(100% - 210px) !important;
     }
     /* Scrubber width (when timers are on) - with audio-tracks/captions */
-    .has-extra-tracks .playkit-player .playkit-seek-bar {
+    .has-captions .playkit-player .playkit-seek-bar {
```

Comment:
for readability, i think it's better to either write it as

.has-captions {
   &.playkit player .playkit-seek-bar {
      ...
   }
}

or 

.has-captions .playkit-player .playkit-seek-bar {
  ...
}

.has-captions .playkit-player .hide-fullscreen .playkit-seek-bar {
   ...
}

Reply:
changed, can you check if look better?

Reply:
yeah, looks good. i would love for it to be hierarchical, but this is a smaller change and in my opinion already more readable